### DISCUSSION TITLE
How to give image width as percentage % instead of pixels px

### DISCUSSION THREAD

**Discussion by jay-softsages** [mjmlio/mjml#2620] [Q&A]
Hi,

I want to give an image a percentage instead of a specific width in pixels. I am getting the image width as a percentage from the backend and if I give a width in pixels then the image is not responsive in the email. 

Is there any solution to this? Please help!

**Comment by iRyusa** [comment]
We don't support image in percent because of Outlook. Not so much we can do on our side.
